lib/stages/pulsarPostProcess.cpp

```diff
@@ -85,19 +85,28 @@ void pulsarPostProcess::fill_headers(unsigned char* out_buf, struct VDIFHeader*
           time_now->tv_nsec);
     for (uint i = 0; i < _num_packet_per_stream; ++i) { // 16 or 80 frames in a stream
         uint64_t fpga_now = (fpga_seq_num + _timesamples_per_pulsar_packet * i);
-        vdif_header->seconds = time_now->tv_sec - unix_offset;
-        vdif_header->data_frame =
+        psr_header->seconds = time_now->tv_sec - unix_offset;
+        psr_header->data_frame =
             (time_now->tv_nsec / 1.e9) / (_timesamples_per_pulsar_packet * fpga_s);
 
         for (uint f = 0; f < freqloop; ++f) {
-            vdif_header->thread_id = thread_ids[f];
+            // Before looping over pulsar beams, load freq labels into header.
+            // As of 7 September 2020, load one of four labels into thread_id, 
```

Comment:
I'm hesitant to include dates in comments.  The time this really changes is when this version code is run on site.  We should rely on the version tracking in `git` and `comet` and not comments to know when things changed.

Reply:
agreed totally, so i've removed this comment.